kafka-client/src/main/java/dev/responsive/kafka/api/stores/ResponsiveSessionParams.java

```diff
@@ -75,10 +76,6 @@ public long numSegments() {
     return this.numSegments;
   }
 
-  public long gracePeriodMs() {
```

Comment:
Why get rid of this? I know it's unused, just thought it would be nice to leave getters for users. (I'm also fine with removing it if you want to, just wanted to explain why it's there despite being unused)

Reply:
the `DslSessionParams` only specifies retention, not grace+inactivity, meaning I can't compute the grace period in the new constructor